index.html

```diff
@@ -22,6 +22,12 @@
                 background-color: red;
             }
 
+            button.page-link:focus {
+                box-shadow: none;
+                outline: 5px auto Highlight;
```

Comment:
I'm trying to find the documentation for `Highlight` in developer.mozilla, but can't seem to find it, could you link me to a reference that documents how this works?

it seems to be working well by the way!

Reply:
https://css-tricks.com/copy-the-browsers-native-focus-styles/
It seems like it's just the default Firefox property. Only supported by that browser.

Reply:
well dang... that's quite a find!

Just a note for the future; normally if this was a browser exclusive feature and it was mostly undocumented, it wouldn't be going here, because mozilla warns that features like this can be removed at anytime by a browser vendor.

Since this is literally for firefox and firefox doesn't comply with the highlighting normally, I think it's worth keeping in, since this is such a really weird edgecase. I'm glad you put the reference here, if someone stops by this PR somehow and sees this implementation, they'll run into this conversation too.

The final iteration looks like this for anyone viewing the PR in the future
![image](https://github.com/iSlammedMyKindle/lemonbotHomepage/assets/108752129/d1646701-89ac-448e-8ae8-7f96b8d54897)

Again, it's not documented and mozilla _could_ remove it, but we have no idea. I'll approve this PR now that there is full coverage for chrome & firefox browsers. Not sure what this looks like on safari, but I'll test that tomorrow after it deploys. (it will likely work given the `-webkit` property)

Thanks @MiguelHigueraDev ! It looks great.

Reply:
No problem. I'll try to replicate the same styling in another PR in the future so it's browser independent and you don't have to worry about it breaking. But I think it's not gonna break in a long time.